react/react1/week3/react1-week3/src/components/TodoList.jsx

```diff
@@ -0,0 +1,39 @@
+import TodoListRow from "./TodoListRow";
```

Comment:
just realized that this code could be on the main component, in React, when we aren't using libraries, we put the functions used in the whole app on the top of the components tree.